From 68374cfa3a0e374d1d859d20c0360f4d46abfa4d Mon Sep 17 00:00:00 2001
From: Fernando Reyna <fernando.m.reyna.diaz@intel.com>
Date: Wed, 17 Feb 2016 11:56:29 -0600
Subject: [PATCH 3/6] OV2680: Defined VIDEO_OV2680=m in x86_64_defconfig

OV2680 was defined because of CHT T3 MRD requires that sensor.

Change-Id: I2f28d1fe48cc93e28b50952ee840e75d49261947
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-20600
Signed-off-by: Fernando Reyna <fernando.m.reyna.diaz@intel.com>
Reviewed-on: https://android.intel.com:443/471769
---
 arch/x86/configs/x86_64_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/x86/configs/x86_64_defconfig b/arch/x86/configs/x86_64_defconfig
index 3706882..77010e4 100644
--- a/arch/x86/configs/x86_64_defconfig
+++ b/arch/x86/configs/x86_64_defconfig
@@ -2518,7 +2518,7 @@ CONFIG_MEDIA_SUBDRV_AUTOSELECT=y
 # CONFIG_VIDEO_GC0310 is not set
 # CONFIG_VIDEO_GC2155 is not set
 # CONFIG_VCM_DW9714 is not set
-# CONFIG_VIDEO_OV2680 is not set
+CONFIG_VIDEO_OV2680=m
 # CONFIG_VIDEO_OV5648 is not set
 
 #
-- 
1.9.1

